src/libraries/System.Text.Json/src/System/Collections/Generic/ValueListBuilder.Pop.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Collections.Generic
+{
+    /// <summary>
+    /// These public methods are required by RegexWriter.
```

Comment:
Common seems specific to a different source generator.

Reply:
this file will go away once https://github.com/dotnet/runtime/pull/71652 is merged.

Reply:
SGTM